roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -131,10 +132,25 @@
                   path: /spec/nodeTemplate/ansible/ansibleVars/edpm_container_registry_insecure_registries
                   value: ["{{ content_provider_registry_ip }}:5001"]
           {% endif %}
-
+          {% cifmw_edpm_deploy_baremetal_repo_setup_override | bool %}
+                - op: add
+                  path: /spec/nodeTemplate/ansible/ansibleVars/edpm_bootstrap_command
+                  value: |
+                    curl -O http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
```

Comment:
this is internal URI - we shouldn't expose it.

Reply:
This project doesn't expose internal things - and getting a downstream-only resource file is usual in here. You may think "it's just uri", right. but it provides far more data than you may think about internal layout. Using parameters and providing them via a config file is easy enough at this point.